src/audio/host-zephyr.c

```diff
@@ -449,8 +449,20 @@ static int host_copy_normal(struct host_data *hd, struct comp_dev *dev, copy_cal
 	comp_dbg(dev, "host_copy_normal()");
```

Comment:
On the commit message, the git commit prefix should refer to the component changed, i.e. here "audio: host: ". 

Could you explain a bit in which case this happens. "src" -> resampler case?

Reply:
Kai, thanks for feedback. 
Regarding the commit message, I will make a correction to specify the component as "audio: host."

Regarding the scenario in which this could happen:
This condition typically occurs at the end of a stream, where the last data chunk does not meet the condition for reload DMA buffer (e.g., large buffers), leaving a non-zero partial_size.